conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -36,6 +37,10 @@
 
     private static final ImmutableList<BiFunction<String, HttpHandler, HttpHandler>> WRAPPERS =
             ImmutableList.<BiFunction<String, HttpHandler, HttpHandler>>of(
+            // Allow the server to configure UndertowOptions.DECODE_URL = false to allow slashes in parameters.
+            // Servers which do not configure DECODE_URL will still work properly except for encoded slash values.
+            (endpoint, handler) -> new URLDecodingHandler(handler, "UTF-8"),
```

Comment:
the javadoc of URLDecodingHandler says:

```
 * If you are using this handler you must set the {@link io.undertow.UndertowOptions#DECODE_URL} parameter to false.
```
, we'll do that in the server, ja?

Reply:
That is correct, however the comment isn't quite right because this handler will no-op to the next handler in the chain if DECODE_URL is not configured.

Reply:
ok. so both of these two handlers would be no-op in that case. can you clarify this in the comment?